rust/cli/src/commands/web_proof.rs

```diff
@@ -90,6 +105,24 @@ pub(crate) async fn webproof_fetch(args: WebProofArgs) -> anyhow::Result<()> {
     Ok(())
 }
 
+pub(crate) fn webproof_debug(args: WebProofDebugArgs) -> anyhow::Result<()> {
+    let web_proof_json = match args.json {
+        Some(web_proof_json) => web_proof_json,
+        None => {
+            #[allow(clippy::expect_used)]
+            let path = args
+                .file
+                .expect("clap guarantees this is defined if args.json is None");
```

Comment:
This `expect` is guaranteed to be "unreachable" by clap's `ArgGroup` defined as `ArgGroup::new("input").required(true).args(&["json", "file"])`.